test/common/upstream/health_check_fuzz.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package test.common.upstream;
+
+import "validate/validate.proto";
+import "test/fuzz/common.proto";
+import "envoy/config/core/v3/health_check.proto";
+
+message Respond {
+  test.fuzz.Headers headers = 1;
+  string status = 2 [(validate.rules).string.pattern = "[0-9]+$*"];
+  //If there's a second host, this will determine which host, if not ignore
+  bool respond_on_second_host = 3;
+}
+
+message StreamCreate {
+  bool create_stream_on_second_host = 1;
```

Comment:
nit: why not simplify this and respond_on to be `second_host`? the message name contains the `stream_create`/`respond` information